at_client/src/main/java/org/atsign/client/cli/Register.java

```diff
@@ -0,0 +1,45 @@
+package org.atsign.client.cli;
+
+import java.util.Scanner;
+import org.atsign.client.util.RegisterUtil;
+
+public class Register {
+    public static void main(String[] args) throws Exception {
+        if (args.length == 0) {
+            System.err.println("Usage: Register <email@email.com>");
+            System.exit(1);
+        }
+
+        String email = args[0];
+        String otp;
+        String validationResponse;
+        String cramSecret;
+
+        Scanner scanner = new Scanner(System.in);
+        Onboard onboardCli = new Onboard();
+        RegisterUtil registerUtil = new RegisterUtil();
+        String atsign = registerUtil.getFreeAtsign();
+        System.out.println("Your atsign is: " + atsign);
```

Comment:
Pls check the code written by Gary on the usage of System.out.print statements vs logs

Reply:
I haven't yet adopted a logging framework.